Patch by Robert Scheck <redhat@linuxnetz.de> for iptraf >= 2.7.0 to set the 
correct paths to some iptraf specific files in the man pages.

This patch solves bug #128476

--- iptraf-2.7.0/Documentation/iptraf.8		2002-03-18 09:05:19.000000000 +0100
+++ iptraf-2.7.0/Documentation/iptraf.8.doc	2004-07-23 15:47:59.000000000 +0200
@@ -86,7 +86,7 @@
 
 .SH FILES
  /var/log/iptraf/*.log - log file
- /var/local/iptraf/* - important IPTraf data files
+ /var/lib/iptraf/* - important IPTraf data files
 
 .SH SEE ALSO
  Documentation/* - complete documentation written by the author
--- iptraf-2.7.0/Documentation/rvnamed.8	2000-03-13 13:03:59.000000000 +0100
+++ iptraf-2.7.0/Documentation/rvnamed.8.doc	2004-07-23 15:49:01.000000000 +0200
@@ -11,7 +11,7 @@
 This program is only used by iptraf and, therefore, is useless alone.
 
 .SH FILES
- /var/local/iptraf/rvnamed.log - log file
+ /var/log/iptraf/rvnamed.log - log file
 
 .SH SEE ALSO
 README.rvnamed - documentation from the author
--- iptraf-3.0.0/Documentation/config.html.orig	2005-08-04 09:16:14.000000000 +0200
+++ iptraf-3.0.0/Documentation/config.html	2006-01-30 15:17:48.000000000 +0100
@@ -82,7 +82,7 @@
   main menu. The configuration is stored in the
   <TT
 CLASS="FILENAME"
->/var/local/iptraf/iptraf.cfg</TT
+>/var/lib/iptraf.cfg</TT
 > file. If the file is not found, IPTraf uses
   the default settings. Any changes to the configuration immediately get
   stored in the configuration file.</P
